src/wizards/generateProject/generationWizard.ts

```diff
@@ -58,14 +58,36 @@ export async function generateProjectWizard() {
     });
 
     state.buildTool = (await input.showQuickPick({
-      title: 'Quarkus Tools',
+      title: INPUT_TITLE,
       step: input.getStepNumber(),
       totalSteps: state.totalSteps,
       placeholder: 'Pick build tool',
       items: quickPickItems,
       activeItem: quickPickItems[0]
     })).label;
 
+    return (input: MultiStepInput) => inputPlatformVersion(input, state);
+  }
+
+  async function inputPlatformVersion(input: MultiStepInput, state: Partial<ProjectGenState>) {
+
+    const quickPickItems: PlatformVersionPickItem[] = await getCodeQuarkusApiPlatforms();
+
+    // Place the recommended option first in the quick pick list
+    quickPickItems.sort((x: PlatformVersionPickItem, y: PlatformVersionPickItem) => {
+      return (x.recommended === y.recommended) ? (x.quarkusCoreVersion < y.quarkusCoreVersion ? -1 : 1) : x.recommended ? -1 : 1;
```

Comment:
quarkusCoreVersion is a string so this comparison won't work how you think. https://www.typescriptlang.org/play?#code/MYewdgziA2CmB00QHMAUAiAjABnr76ABADyHoBMee6AlEA

I'd also just update the comment above to mention that we now sort by recommended, and then use version..

Reply:
https://github.com/redhat-developer/vscode-quarkus/pull/394#discussion_r714089771